src/hooks/navigation/usePreventGoBack.ts

```diff
@@ -1,12 +1,27 @@
 import { useNavigation } from '@react-navigation/native'
 import { useEffect } from 'react'
 import { useTranslation } from 'react-i18next'
-import { Alert } from 'react-native'
+import { Alert, Platform } from 'react-native'
 
 export const usePreventGoBack = (shouldPrevent = true): void => {
   const { t } = useTranslation()
   const navigation = useNavigation()
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const alertPolyfill = (title: string, description: string, options: any) => {
+    const result = window.confirm([title, description].filter(Boolean).join('\n'))
+
+    if (result) {
+      const confirmOption = options.find(({ style }: { style: string }) => style !== 'cancel')
+      confirmOption && confirmOption.onPress()
+    } else {
+      const cancelOption = options.find(({ style }: { style: string }) => style === 'cancel')
+      cancelOption && cancelOption.onPress()
+    }
+  }
```

Comment:
Move it utils, you can do something like this:

```tsx
import { Alert as RNAlert, Platform } from 'react-native'


  const alertPolyfill = (title: string, description: string, options: any) => {
    const result = window.confirm([title, description].filter(Boolean).join('\n'))

    if (result) {
      const confirmOption = options.find(({ style }: { style: string }) => style !== 'cancel')
      confirmOption && confirmOption.onPress()
    } else {
      const cancelOption = options.find(({ style }: { style: string }) => style === 'cancel')
      cancelOption && cancelOption.onPress()
    }
  }
  
  const Alert = {
     ...RNAlert,
     alert: Platform.OS === 'web' ? alertPolyfill : RNAlert.alert
  }

```

Thanks to that, we can use this alert in all components